src/Servers/IIS/Directory.Build.props

```diff
@@ -1,6 +1,13 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+  
   <PropertyGroup>
     <KestrelSharedSourceRoot>$(MSBuildThisFileDirectory)..\Kestrel\shared\</KestrelSharedSourceRoot>
+    <HostArchDir>$(Platform)</HostArchDir>
```

Comment:
nit, but I prefer `HostArch` to `HostArchDir`

Reply:
Looks better to me too, will go with that